jenkins-scripts/dsl/ros1_ign_bridge.dsl

```diff
@@ -33,7 +33,8 @@ bridge_packages.each { pkg ->
     }
 
     parameters {
-      stringParam("PACKAGE","$pkg_dashed","Package name to be built")
+        stringParam("PACKAGE","$pkg_dashed","Package name to be built")
+        stringParam("IGNITION_VERSION", '', 'Ignition release supported in the binaries')
```

Comment:
Would it be a good idea to have a default value here?

Reply:
I don't think that we have a good candidate given that it will vary with time and we probably want it to fail if unset instead of running a wrong distro.